apps/ledger-live-desktop/src/renderer/screens/settings/sections/Accounts/BlacklistedTokens.test.tsx

```diff
@@ -0,0 +1,207 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from "react";
+import { render, screen, waitFor } from "tests/testSetup";
+import { fireEvent } from "@testing-library/react";
+import BlacklistedTokens from "./BlacklistedTokens";
+import { CryptoCurrency, TokenCurrency } from "@ledgerhq/types-cryptoassets";
+
+const mockSync = jest.fn();
+const mockDispatch = jest.fn();
+const mockFindTokenById = jest.fn();
+
+jest.mock("@ledgerhq/live-common/bridge/react/index", () => ({
+  useBridgeSync: () => mockSync,
+}));
+
+jest.mock("@ledgerhq/live-common/bridge/crypto-assets/index", () => ({
+  getCryptoAssetsStore: () => ({
+    findTokenById: mockFindTokenById,
+  }),
+}));
+
+jest.mock("react-redux", () => ({
```

Comment:
I think you can avoid this mock and useSelector, and use the 2nd param of render in each test instead eg `render(<BlacklistedTokens />, { initialState: { settings: { blacklistedTokenIds: [] } } });`